src/index.js

```diff
@@ -0,0 +1,19 @@
+import './style.css';
+
+import {
+  memory, root, ul,
+} from './storage.js';
+
+class LeaderBoard {
+  display = (memory, root, ul) => {
+    memory.forEach((currentItem, index) => {
+      ul.innerHTML += `
+      <li id='${index + 1}'>${currentItem.name}: ${currentItem.score}</li>
+      `;
+    });
+    root.append(ul);
+  }
+}
```

Comment:
- [ ] Good job creating the application layout 👍  however as I already mentioned part of the goal for this assignment is to get familiar with ES6 modules to write modular JavaScript with proper ES6 syntax. therefore I mentioned that this should be an arrow function if you replace the class with an arrow function you're ready to go.
if you need any help solving this issue you can reach out to me on slack as well if you want.

Reply:
Thanks.